src/fs/rock/RockIoRequests.h

```diff
@@ -23,23 +24,31 @@ class ReadRequest: public ::ReadRequest
     CBDATA_CLASS(ReadRequest);
 
 public:
-    ReadRequest(const ::ReadRequest &base, const IoState::Pointer &anSio);
+    ReadRequest(const ::ReadRequest &base, const IoState::Pointer &anSio, const IoXactionId);
```

Comment:
For consistency (with other parameters) sake:

```suggestion
    ReadRequest(const ::ReadRequest &base, const IoState::Pointer &anSio, const IoXactionId anId);
```

or for simplicity/DRY sake:

```suggestion
    ReadRequest(const ::ReadRequest &, const IoState::Pointer &, const IoXactionId);
```

Reply:
Done the second suggestion.